spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/IntegrationNode.java

```diff
@@ -35,12 +39,20 @@
 
 	private final String componentType;
 
+	private final Map<String, Object> properties = new HashMap<String, Object>();
+
 	protected IntegrationNode(int nodeId, String name, Object nodeObject, Stats stats) {
 		this.nodeId = nodeId;
 		this.name = name;
 		this.componentType = nodeObject instanceof NamedComponent ? ((NamedComponent) nodeObject).getComponentType()
 				: nodeObject.getClass().getSimpleName();
 		this.stats = stats;
+		if (nodeObject instanceof IntegrationObjectSupport) {
```

Comment:
M-m-m. Seems for me we still have a Proxy problem here.
OTOH `IOT` is `ExpressionCapable` already. So, no reason to hack classes over reflection.